framework/src/ble_manager/ble_manager_state.c

```diff
@@ -962,7 +964,9 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 			break;
 		}
 		int i;
-		ble_conn_handle data = *(ble_conn_handle *)msg->param;
+		uint32_t value = *(uint32_t *)msg->param;
+		ble_conn_handle data = (ble_conn_handle)(value >> 16);
+		uint8_t reason = (uint8_t)(value & 0xff);
```

Comment:
The local variable reason is not being used in this case. Do we need this?

Reply:
Thanks, you are right. I will remove 'reason'.